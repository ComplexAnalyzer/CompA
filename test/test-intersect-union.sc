def main() {
    a = [1,2,3,4];
    b = [3,4,7,8,9];
    c = a * b;
    print("intersect");
    (0<=i<#c) print(c[i]);
    d = a & b;
    print("union");
    (0<=i<#d) print(d[i]);
}
