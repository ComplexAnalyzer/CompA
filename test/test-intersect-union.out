intersect
3
4
union
1
2
3
4
7
8
9
